examples/benchmark/main.go

```diff
@@ -9,9 +9,9 @@ import (
 
 	"github.com/pkg/profile"
 
-	"github.com/optimizely/go-sdk/pkg/client"
-	"github.com/optimizely/go-sdk/pkg/decision"
-	"github.com/optimizely/go-sdk/pkg/entities"
+	"github.com/optimizely/go-sdk/v2/pkg/client"
+	"github.com/optimizely/go-sdk/v2/pkg/decision"
+	"github.com/optimizely/go-sdk/v2/pkg/entities"
```

Comment:
I think this is now the 3rd thing about Go that makes me 🤔 hard: adding these major versions directly in the import. 😕 

I wish this was a situation we could use a `const` to handle...but I get it.

Reply:
....or somehow wrap into a single file all the imports and then re-export as an alias...or something.

Reply:
Is the go-testapp also going to need to be updated for OTel & /v2?

Reply:
Yes, we need to update the testapp otherwise FSC will keep failing